src/main/java/Duke.java

```diff
@@ -1,10 +1,423 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    /** Maximum number of tasks */
+    private static final int MAX_TASK_NUM = 100;
+
+
+    /** Number of tasks in the list */
+    private static Task[] tasks = new Task[MAX_TASK_NUM];
+
+    /** Counter of how many tasks there are */
+    private static int taskCounter = 0;
+
+    /** Scanner to receive input from user */
+    private static final Scanner INPUT = new Scanner(System.in);
+
+    /** User commands available to the user */
+    private static final String CMD_ADD_TODO = "todo";
+    private static final String CMD_ADD_DEADLINE = "deadline";
+    private static final String CMD_ADD_EVENT = "event";
+    private static final String CMD_DONE = "done";
+    private static final String CMD_LIST = "list";
+    private static final String CMD_EXIT = "bye";
+    private static final String CMD_HELP = "help";
+    private static final String CMD_CLEAR_WINDOW = "clear";
+
+    /** Easter egg */
+    private static final String CMD_DEBUG = "debug";
+
+    /** Flag to check whether the main program should be running or not */
+    private static boolean isProgramRunning = true;
+
+    /** Flag to avoid printing unnecessary output when debugging */
+    private static boolean inDebugMode = false;
+
+
+    /**
+     * Prints horizontal line for chat bot.
+     */
+    public static void printLine (boolean hasNewline) {
```

Comment:
Consider changing function name to printHorizontalLine() to be more specific about what the function does

Reply:
That's true, I'll try to have better names for my functions and variables in the future 👍